rust/processor/src/db/postgres/migrations/2024-08-28-015552_fa_migration_fix_2/up.sql

```diff
@@ -0,0 +1,15 @@
+-- Your SQL goes here
+ALTER TABLE current_unified_fungible_asset_balances_to_be_renamed
+ADD COLUMN IF NOT EXISTS asset_type VARCHAR(1000) NOT NULL GENERATED ALWAYS AS (COALESCE(asset_type_v1, asset_type_v2)) STORED;
```

Comment:
We need to redo this migration because it has already been applied to the mainnet and testnet db.